src/index.js

```diff
@@ -258,7 +258,33 @@ class App extends Component {
         hideControls={this.state.hideControls}
         data={this.state.data}
         layout={this.state.layout}
-        config={{ displayModeBar: false }}
+        config={{
+          displayModeBar: true,
```

Comment:
This seems like the right setting so it only shows on hover.

```suggestion
          displayModeBar: 'hover',
```